org.jacoco.core.test/src/org/jacoco/core/test/validation/SourceTest.java

```diff
@@ -76,12 +76,4 @@ public void testDuplicateTag() throws IOException {
 		new Source(new StringReader(src));
 	}
 
-	@Test
-	public void testGetSourceFor() throws IOException {
-		final Source s = Source.getSourceFor("src", SourceTest.class);
-		// Here we are. $line-testGetSourceFor$
-		final String l = s.getLine(s.getLineNumber("testGetSourceFor"));
-		assertTrue(l, l.contains("Here we are."));
```

Comment:
Damn, I've overlooked this: There is another unused import for `assertTrue` now.

Reply:
@marchof fixed, also checked that there is no other unused imports in modified files :wink:

Reply:
👍